micro-ui/web/micro-ui-internals/packages/modules/core/src/components/TopBarSideBar/SideBar/EmployeeSideBar.js

```
@@ -58,34 +52,16 @@ const EmployeeSideBar = () => {
-            return acc[key]; // Return the nested object for the next iteration.
-          }
-        }, hierarchicalMap);
-        mergeObjects(configEmployeeSideBar, hierarchicalMap);
-      } else if (
-        checkMatch(t(`ACTION_TEST_${index?.toUpperCase()?.replace(/[ -]/g, "_")}`), search) ||
-        checkMatch(t(Digit.Utils.locale.getTransformedLocale(`ACTION_TEST_${item?.displayName}`)), search)
-      ) {
-        const keys = item.path.split(".");
+      if (item?.path !== "") {
+        const keys = item?.path?.split(".");
         let hierarchicalMap = {};
```

Biome

[error] This let declares a variable that is only assigned once.

'hierarchicalMap' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)